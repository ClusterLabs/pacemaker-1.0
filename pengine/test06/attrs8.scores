Allocation scores:
native_color: rsc1 allocation score on node1: 200
native_color: rsc1 allocation score on node2: 400
