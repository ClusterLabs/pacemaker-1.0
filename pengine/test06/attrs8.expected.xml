<?xml version="1.0" encoding="UTF-8"?>
<cib admin_epoch="0" epoch="1" num_updates="1" dc-uuid="uuid1" have-quorum="false" remote-tls-port="0" validate-with="pacemaker-1.0">
  <configuration>
    <crm_config>
      <cluster_property_set id="cib-bootstrap-options"><nvpair id="nvpair" name="no-quorum-policy" value="ignore"/></cluster_property_set></crm_config>
    <nodes>
      <node id="uuid1" uname="node1" type="member"/>
      <node id="uuid2" uname="node2" type="member"/>
    </nodes>
    <resources>
      <primitive id="rsc1" class="heartbeat" type="apache"><meta_attributes id="primitive-rsc1.meta"/></primitive>
    </resources>
    <constraints>
      <rsc_location id="run_rsc1" rsc="rsc1">
	<rule id="pingd-rule" score-attribute="pingd">
	  <expression attribute="pingd" operation="defined" id="expression"/>
	</rule>
      </rsc_location>
    </constraints>
  </configuration>
  <status>
    <node_state id="uuid1" ha="active" uname="node1" crmd="online" join="member" expected="member" in_ccm="true">
      <transient_attributes id="uuid1">
	<instance_attributes id="instance_attributes"><nvpair id="nvpair" name="probe_complete" value="true"/><nvpair id="nvpair" name="pingd" value="200"/></instance_attributes>
      </transient_attributes>
    </node_state>
    <node_state id="uuid2" ha="active" uname="node2" crmd="online" join="member" expected="member" in_ccm="true">
      <transient_attributes id="uuid2">
	<instance_attributes id="instance_attributes"><nvpair id="nvpair" name="probe_complete" value="true"/><nvpair id="nvpair" name="pingd" value="400"/></instance_attributes>
      </transient_attributes>
    </node_state>
  </status>
</cib>
